scripts/invalidate_unmodified_studies.py

```
@@ -0,0 +1,101 @@
+#
+# Example:
+#   python unmount_unmodified_studies.py P365D
+#   python unmount_unmodified_studies.py P365D --batch-size 10 --dry-run
+#
+
+#
+# @author Hugo Marcellin <hugo.marcellin_externe at rte-france.com>
+#
+
+def get_unmodified_studies(duration):
+    response = requests.get(constant.GET_UNMODIFIED_DIRECTORY_ELEMENTS, params={"elementType": "STUDY", "duration": duration})
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)